### DISCUSSION TITLE
Missing Unicode in Julia Fenced Code in `pdf` format

### DISCUSSION THREAD

**Discussion by agerlach** [quarto-dev/quarto-cli#4165] [Q&A]
I have missing unicode characters in my pdf output. This issue and a fix is presented [here](https://github.com/quarto-dev/quarto-cli/discussions/3076). However, this requires change the font for the entire document. Is there a way to change only the font used for the fenced code?

**Comment by baptiste** [comment]
`monofont: "JuliaMono"` should only affect the code parts, not the main text which can be set in another font (ideally pairing well, aesthetically speaking). Do you mean you only want _some_ code to be in JuliaMono?

````
---
title: "test of julia"
author: "SJE"
execute:
  echo: true
format:
  beamer:
    keep-tex: true
    monofont: "JuliaMono"
    mainfont: "Minion 3"
---

## Test slide

Some text with unicode `>> α ϐ ∈ <<` here

```{{julia}} 
for i ∈ 20:22 println(i) end
```

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
````
<img width="887" alt="Screenshot 2023-02-01 at 07 55 33" src="https://user-images.githubusercontent.com/18970/215855901-fe2a5290-b446-4a26-932a-ca001fb69bab.png">

**Comment by agerlach** [comment]
I see. When I saw that example in linked post, I didn't catch that the `>> α ϐ ∈ <<` was in ticks. So, I thought it was displaying the unicode with the main font too, which doesn't work with the default main font. So, that made me believe it was changing the font everywhere. Thanks for helping clear it up for me.

**Reply by baptiste** [reply]
I thought that might be the case, glad it helped. 

By the way, I quite like the [Iosevka family](https://typeof.net/Iosevka/) of coding fonts, and they have broad glyph coverage.

**Reply by agerlach** [reply]
Cool Thanks
